android/app/capacitor.build.gradle

```diff
@@ -9,7 +9,7 @@ android {
 
 apply from: "../capacitor-cordova-android-plugins/cordova.variables.gradle"
 dependencies {
-    implementation project(':capacitor-pusher-beams')
```

Comment:
@ronakamani i think you accidentally removed our pusher-beams plugin.

Reply:
@trtshen, so weird. I'll fix it now.